Src/Penneo/(Model)/Document.cs

```diff
@@ -31,12 +32,21 @@ public Document(CaseFile cf)
             CaseFile = cf;
         }
 
-        public Document(CaseFile cf, string title, string pdfFilePath)
+        public Document(CaseFile cf, string title, string pdfFile)

```

Comment:
```suggestion
        public Document(CaseFile cf, string title, string pdfFileContentOrPath)
```

And maybe add some documentation for our integration partners ;)